platforms/web/src/components/DemoConfigDialog/DemoConfigDialog.tsx

```diff
@@ -170,7 +155,7 @@ const DemoConfigDialog = ({ query }: { query: BootstrapData }) => {
             helpLink={'https://docs.jwplayer.com/platform/docs/ott-create-an-app-config'}
             error={typeof state.error === 'string' ? undefined : state.error}
           >
-            <form method={'GET'} target={'/'}>
+            <form method={'GET'} target={PATH_HOME}>
```

Comment:
the `target` attribute can be discarded. In our case, it does nothing.
When used, it should contain other values as described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#target

Reply:
You're right, it should be `_self`, which is the default.

Reply:
That also required a snapshot update.